editor/plugins/tile_set_editor_plugin.cpp

```diff
@@ -283,13 +283,15 @@ void TileSetEditorPlugin::make_visible(bool p_visible) {
 
 	if (p_visible) {
 		tileset_editor->show();
+		tileset_editor->menu->show();
```

Comment:
You shouldn't have to show it manually.. I guess you're not adding it properly as child of `tileset_editor`?

Reply:
Making it a child of the TilesetEditor was what it caused to go out of bounds. Also fixed the compilation error.

Reply:
It feels hacky this way, but well if it works..

Reply:
On the topic of the TilesetEditor, is the dock used at all? Because I'm trying everything with the Tile Set resource but the dock doesn't seem to participate in anything.
![screenshot1](https://user-images.githubusercontent.com/30739239/34565535-521e86e6-f142-11e7-9ab9-8ba713f55dd4.png)